packages/block-editor/src/components/list-view/index.js

```diff
@@ -205,6 +207,7 @@ function ListView(
 				blockDropTarget={ blockDropTarget }
 			/>
 			<TreeGrid
+				id={ id }
```

Comment:
Note that this component accepts arbitrary props and spreads them onto the underlying `<table>` element as `...props`. Thus we are not adding a new prop to `<TreeView>`.